libs/ui/src/diagnostics/configuration_section.tsx

```diff
@@ -18,6 +18,7 @@ export interface ConfigurationSectionProps {
   electionPackageHash?: string;
   expectPrecinctSelection?: boolean;
   markThresholds?: MarkThresholds;
+  omitConfigSectionBallotStyles?: boolean;
```

Comment:
Would it be simpler if we combine the ballot styles info into one section that's used across all machines? I.e. remove the ballot style info from the config section and add the new ballot styles section to all readiness reports?

Reply:
Ah, I like the idea of having consistency across the apps for this piece. Thinking through whether that would involve filtering down to only the relevant ballot styles for a precinct when rendering on a single-precinct machine - will add this to an open thread about this task and follow up with a change if needed.